db/migrations/202504231037_add_indexes.go

```diff
@@ -0,0 +1,45 @@
+package migrations
+
+import (
+	_ "embed"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+var _202504231037_add_indexes = &gormigrate.Migration{
+	ID: "202504231037_add_indexes",
+	Migrate: func(db *gorm.DB) error {
+
+		if err := db.Transaction(func(tx *gorm.DB) error {
+
+			if err := tx.Exec(`
+DROP INDEX IF EXISTS idx_transactions_app_id;
```

Comment:
These are not being used currently?

Reply:
@im-adithya how did you confirm the indexes are not used?

Reply:
I confirmed by seeing the possible queries from `transaction_service.go` etc.

For example, we never filter based on `idx_transactions_created_at`, `idx_transactions_settled_at` or `idx_transactions_request_event_id` so I removed those (and also as mentioned above, we don't need single indexes as most of our queries are chained involving more than one column so single col indexes won't help anyways)